tests/functional/detection_status/test_detection_status.py

```diff
@@ -64,6 +64,10 @@ def setUp(self):
 int main()
 {
   int i = 1 / 0;
+
+  sizeof(42);
```

Comment:
Could you add a test case on solving a bug to see, that detection status is working properly.